src/include/OutputFieldsDefault.h

```diff
@@ -225,6 +226,7 @@ class OutputFieldsDefault : public OutputFieldsParams
       prof_stop(pr_moment);
     }
 
+
```

Comment:
I generally would avoid unnecessary whitespace changes. In the future, this should be taken care of by clang-format for the most part. If there is a reason to do lots of whitespace changes, I'd prefer them to be in a separate commit. Obviously, for a single line it's a non-issue.

Reply:
You added some more unwanted whitespace changes. clang-format actually would undo those. But don't worry about them for now.

Reply:
However, I was going to ask whether you tested it, but I can actually see that you didn't, because it won't compile. But more important would be to make sure that it actually works as desired.

I think there's a bug which came from my suggestion. If these is restarted, ie., `timestep != 0`, not only will the output be skipped (which is desired), it'll also not set `first_time = false`. So I think this needs two levels of if.